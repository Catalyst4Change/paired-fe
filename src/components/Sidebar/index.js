import React, { Component } from 'react';
import { NavLink, Link } from 'react-router-dom';

export default class Sidebar extends Component {
  render() {
    return (
      <aside className='Sidebar'>
        <Link to='/'>
          <h1>Paired</h1>
        </Link>
        <NavLink to='/schedule'>View Schedule</NavLink>
        <NavLink to='/book-pairing'>Book a Pairing</NavLink>
      </aside>
    );
  }
}
