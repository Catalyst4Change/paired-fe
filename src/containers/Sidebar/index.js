import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

export default class Sidebar extends Component {
  render() {
    return (
      <aside className="Sidebar">
        <NavLink to="/schedule">View Schedule</NavLink>
        <NavLink to="/book-pairing">Book a Pairing</NavLink>
      </aside>
    );
  }
}
