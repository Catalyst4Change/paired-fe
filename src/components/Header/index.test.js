import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './index';

describe('Sidebar', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper).toMatchSnapshot();
  });
});