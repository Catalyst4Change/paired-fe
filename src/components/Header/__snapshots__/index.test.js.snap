// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should match the snapshot 1`] = `
<aside
  className="Sidebar"
>
  <Link
    to="/"
  >
    <h1>
      Paired
    </h1>
  </Link>
  <NavLink
    to="/schedule"
  >
    View Schedule
  </NavLink>
  <NavLink
    to="/book-pairing"
  >
    Book a Pairing
  </NavLink>
</aside>
`;
